### PR TITLE
Added cache duration schedule for latest tweets

### PR DESCRIPTION
**Pull request by grafxflow**
1.0.6: Added cache duration schedule for latest tweets (default 10
mins).

### CODE REVIEW

**Comment by LukeTowers** [comment]
Indentation in spaces please, not tabs

**Comment by grafxflow** [comment]
Should be fine now.

**Comment by grafxflow** [comment]
Also change the default 10 minutes to the original 2 minutes.

File: classes/TwitterClient.php [hunk 1]
```diff
@@ -95,8 +97,10 @@ public function getLatestTweet($params)
             $text = preg_replace('/\#\w+/', '<span class="tag">$0</span>', $text);
             $message['text_processed'] = $obj->urlsToLinks($text);
         }
-
-        Cache::put($cacheKey, serialize($result), 2);
+		
+		$expiresAt = Carbon::now()->addMinutes($params["cache-duration"]);
```
Comment (LukeTowers) [inline classes/TwitterClient.php]: whitespace

Comment (LukeTowers) [reply classes/TwitterClient.php]: Also why are you using Carbon here instead of just passing the cache-duration directly?

**Comment by grafxflow** [comment]
Corrected tab spacing

**Comment by daftspunk** [comment]
Thanks!

**Pull request merged**